installers/Config.hs

```diff
@@ -0,0 +1,134 @@
+{-# LANGUAGE LambdaCase          #-}
```

Comment:
This file should also use `NoImplicitPrelude` for consistency

Reply:
Done.